### PR TITLE
Replaced string based sytem with template literals

### PR DESCRIPTION
**Pull request by a19danla**
Changed current string based system with template literals where the a row of code exceeded 80-100 characters.
Only changed where it increased the readability of the code.

### CODE REVIEW

**Review by a19kimgr — APPROVED** [review]
Looks good, no errors on my side!

**Review by a19klafr — CHANGES_REQUESTED** [review]
Is there a reason

id='" + col + "_" + obj.uid + "'

stays this way on line 521, and not gets changed into

id='${col}_${obj.uid}'

like in blocks above and below this one?

File: DuggaSys/accessed.js [hunk 1]
```diff
@@ -508,18 +508,24 @@ function renderCell(col, celldata, cellid) {
...
 		} else {
-			str = "<div style='display:flex;'><span id='" + col + "_" + obj.uid + "' style='margin:0 4px;flex-grow:1;'>" + obj[col] + "</span></div>";
+			str = `<div style='display:flex;'><span id='${col}_${obj.uid}' style='margin:0 4px;flex-grow:1;'>
+			${obj[col]}</span></div>`;
 		}
 	} else if (col == "class") {
 		var className = obj.class;
 		if (className == null || className === "null") {
 			className = "";
-			str = "<div class='access-dropdown' id='" + col + "_" + obj.uid + "'><div style='color:#808080'> None"+className+"</div><img class='sortingArrow' src='../Shared/icons/desc_black.svg'/>" + makedivItem(className, filez['classes'], "class", "class") + "</div>";
```
[the remark this answers is not in the document]
Comment (a19danla) [reply DuggaSys/accessed.js:519]: I must've missed that line. Will change it to match the rest!

**Review by a19klafr — CHANGES_REQUESTED** [review]
€{col}_${obj.uid}
wrong sign in beginning of this block, line 521

**Review by a19klafr — COMMENTED** [review]
Everything is in order

**Review by a19klafr — APPROVED** [review]
Everything looks good

**Pull request merged**